display_manager.py

```diff
@@ -27,14 +27,27 @@ class DisplayManager(metaclass=Singleton):
     FOV_LIGHT_WALLS = True
     FOV_RADIUS = 10
 
+    BAR_WIDTH = 20
+    PANEL_HEIGHT = 6
+    PANEL_Y = SCREEN_HEIGHT - PANEL_HEIGHT
+
+    MSG_X = BAR_WIDTH + 2
+    MSG_WIDTH = SCREEN_WIDTH - BAR_WIDTH - 2
+    MSG_HEIGHT = PANEL_HEIGHT - 1
+
+    game_msgs = []
+
+    # TODO: give consoles a better name
     tdl.set_font('lucida10x10_gs_tc.png', greyscale=True, altLayout=True)
-    console = tdl.Console(SCREEN_WIDTH, SCREEN_HEIGHT)
+    console = tdl.Console(SCREEN_WIDTH, SCREEN_HEIGHT-4)
```

Comment:
Where does this 4 come from? Why not use `PANEL_HEIGHT` or make it a new constant?

Reply:
whoops